### PR TITLE
Update sshd config recommendations

### PR DESCRIPTION
**Pull request by xrchz**

### CODE REVIEW

**Comment by jclapis** [comment]
@actuallymentor Can you take a look at these and verify?

File: src/guides/node/securing-your-node.md [hunk 1]
```diff
@@ -318,7 +318,7 @@ This is a large file, so you'll have to navigate through it using the arrow keys
 Make the following changes:
 
 1. Uncomment `#AuthorizedKeysFile` if it is commented (by removing the `#` in front of it)
-2. Change `ChallengeResponseAuthentication yes` to `ChallengeResponseAuthentication no`
+2. Change `KbdInteractiveAuthentication yes` to `KbdInteractiveAuthentication no`
```
Comment (jshufro) [inline src/guides/node/securing-your-node.md]: 20.04 still uses ChallengeResponseAuthentication, so we should ideally adjust the instructions to work for both 22.04 and later or 20.04

**Comment by actuallymentor** [comment]
@xrchz could you amend this PR to say "if you are using 20.04 do x and on 22.04 do y" as per @jshufro's comment please?

**Comment by actuallymentor** [comment]
Thanks for the edit @xrchz! Could you please use specific version numbers instead of "older". The `20.04 LTS` schedule means it will be industry standard for a while.

Perhaps add how people can check the `ssh` version relevant to this decision.

**Comment by xrchz** [comment]
I'm not sure that that is worthwihle - the way to check which version you have is just to look at the config file you're editing. The version isn't relevant except to explain why you might be seeing one keyword instead of another.

**Comment by actuallymentor** [comment]
In my experience, node operators are less aware of the nuances of shell/ssh than we would assume. You can perhaps add a `note` that explains this with come context? Most users will skim it, but those that need more context would have it.

**Comment by xrchz** [comment]
I think I'm confused about the flow you're imagining here. What I am thinking is:

- the node operator will get to this point in the guide and see an instruction to change a line in their config file
- if they're on newer ssh it'll be fine and they move on
- if they're on older ssh the line they're supposed to change won't exist and they'll be momentarily confused
- then they'll see the parenthesis noting the old keyword and realise that the line with the old keyword is the one to change instead

There's no need for them to think about exactly which version of Ubuntu they are on.

**Comment by jclapis** [comment]
Amended the note slightly - I think it's time, let's see if people start coming in complaining about `ChallengeResponseAuthentication` now.

**Pull request merged**